.github/workflows/pull-request-review.yml

```
@@ -0,0 +1,46 @@
+      - name: Check CodeRabbit approval using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // List all reviews for the PR
+            const { data: reviews } = await github.rest.pulls.listReviews({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)